docs/user/authentication_and_authorization.md

```diff
@@ -241,7 +244,7 @@ The `subjectInfo` property is present only if other user attributes are known to
 
 ### Add rights on entities for a Stellio User
 
-This endpoint allows an user to give rights on entities it is admin of.
+This endpoint allows an user to give rights on entities it is admin or owner of.
```

Comment:
here also : _This endpoint allows a user to give rights on entities they are admin or owner of._